### ISSUE TITLE
The web app seems to be not working?

### ISSUE THREAD

**Issue by saturosfz** [gigerdo/gazecorrection-js#2]
I am Chinese. When I use the web app, the rendered result seems to be the same as the normal camera shows. In the debug view, the wire view and the track points can be shown, which means that the app is ok. Anyone who succeeded in running the app?

**Comment by gigerdo** [comment]
Make sure to open the face rotation tab and move the rotX slider.

**Comment by saturosfz** [comment]
I move the slider, and I can see the rendered head rotating, but it seems that the web app can only adjust the head pose manually, but the automatic gaze correction is not included. Is is the case? Thank you, gigerdo:)

**Comment by gigerdo** [comment]
Yes that is true, there is no automatic gaze correction. This is actually a very basic implementation that I quickly did in a couple of hours so the performance is quite poor and the results are not very good. It needs a lot of polishing to be useable.
